flow/EncryptUtils.cpp

```diff
@@ -24,6 +24,10 @@
 #include <boost/algorithm/string.hpp>
 #include <boost/format.hpp>
 
+const EncryptCipherDomainName FDB_SYSTEM_KEYSPACE_ENCRYPT_DOMAIN_NAME = "FdbSystemKeyspaceEncryptDomain"_sr;
+const EncryptCipherDomainName FDB_DEFAULT_ENCRYPT_DOMAIN_NAME = "FdbDefaultEncryptDomain"_sr;
+const EncryptCipherDomainName FDB_ENCRYPT_HEADER_DOMAIN_NAME = "FdbEncryptHeaderDomain"_sr;
```

Comment:
Do these need to be standalone types? The memory for the data should be valid without storing them in arenas.

Reply:
This is so that I can insert them into a data structure that accepts Standalone's without copying